src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+
+/**
+ * Command to exit Duke.
+ */
+public class ExitCommand extends Command {
+
+    /**
+     * Constructor for the exit command.
+     */
+    public ExitCommand(){
```

Comment:
Shouldn't there be a white space before the open curly bracket?

Reply:
Yep, fixed it.